src/util/get-config-files.ts

```diff
@@ -32,11 +32,13 @@ export async function getConfigFiles(
     ref: branch,
   });
 
+  const files = [requestedRepoFile];
```

Comment:
I am just moving that `files` array up a bit and also already push files into it. Easy.